feature : script to display a welcome message with today date

Moruald001 opened the pull request:
Feature  #10 
This script  display a welcome message with user full name and today's date

1.  Display a message inviting the user to enter their full name

2.  Wait for user to enter their full name and store it in a variable ( USER_FULLNAME )

3.   Save the current date in a variable ( TODAY ) by adding the appropriate suffix (st, nd, rd, th) for the day: st for 1st, 21st, 
       and 31st ; nd for 2nd and 22nd ; rd for 3rd and 23rd ; and th for all other days, using awk in the format  %A, %B %d %Y 

4.   display a welcome message  with USER_FULLNAME and TODAY

Moruald001 committed:
feature added : script to display a welcome message with today date

centia28 commented:
@Moruald001 I made a mistake. Assignees are persons who works on the task. So you have to update the pull request and add us in the reviewers. You remove us from the assignees and you can add yourself there.
PS: I updated the instructions in Notion.

Moruald001 commented:
@centia28 
I changed all of this, 
it should be okay now

centia28 commented:
> * you did not link the PR with the issue.

You still have to link the PR with the issue.
PS: Check the instructions in Notion to make sure you've respected all the steps.

Moruald001 commented:
![Capture d’écran 2025-03-15 112408](https://github.com/user-attachments/assets/4f199d6c-c70a-41dc-808b-8446df4f1f0c)

But it's already linked here

centia28 commented:
> But it's already linked here
Oh okay my bad. I'll review the PR

centia28 commented on welcome-message-with-date.sh:
```diff
@@ -0,0 +1,11 @@
+echo "what's your full name"
+read USER_FULLNAME
+
+#I save the current date in TODAY by adding the suffix corresponding to the day
```
A bette comment would be
Save the date in the format "$DAY_NAME, $DAY $MONTH $YEAR"

Moruald001 replied:
Okay i will change it

centia28 reviewed (approved):
Overall, good job!

The review process is available here: https://www.notion.so/1-2-4-Collaboration-avec-GitHub-1ae7e6a2c37f81f3a891c72d1f23db22?pvs=4#1b77e6a2c37f80dfa251c03209afdf19

centia28 commented:
The review process is available here: https://www.notion.so/1-2-4-Collaboration-avec-GitHub-1ae7e6a2c37f81f3a891c72d1f23db22?pvs=4#1b77e6a2c37f80dfa251c03209afdf19

Moruald001 committed:
Update welcome-message-with-date.sh

replace the comment in line 7 with "Save the date in the format $DAY_NAME, $DAY $MONTH $YEAR"
